migrations/20250528184917-wclUser.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  up: (queryInterface, DataTypes) => {
+    return Promise.all([
+      queryInterface.createTable("WclUser", {
```

Comment:
ah, this could have been a modification of the user table, instead of a separate table. migrations can do `ALTER TABLE` statements on existing tables created by previous migrations. sorry that i wasn't clear :sweat_smile: 

but if this works, I'm not sure it needs to change. i'm going to pull this + the frontend part and poke around with some private logs to see